README.md

```diff
@@ -43,7 +43,8 @@ npm install -g @asyncapi/generator
 | version | Override the version of your application provided under `info.version` location in the specification file. | No | Version is taken from the spec file. | *Any* ([See Semver versioning](https://semver.org/)) | `1.0.0` |
 | singleFile | Set output into one html-file with styles and scripts inside | No | `false` | `true`,`false` | `true` |
 | outFilename | The filename of the output file. | No | `index.html` | *Any* | `asyncapi.html` |
-| pdf | Generates output HTML as PDF | No | `false` | `true,false` | `false` |
+| pdf | Generates output HTML as PDF | No | `false` | `true,
+| timeout | Timeout (in ms) used to generate the PDF | No | 30000 | >=0 | `false` |false` | `false` |
```

Comment:
It's not a timeout of template, but for pdf generation :) We should use `pdfTimeout` to clarify what it is. Also you have broken md table here.

```suggestion
| pdf | Generates output HTML as PDF | No | `false` | `true`, `false` | `true` |
| pdfTimeout | Timeout (in ms) used to generate the PDF | No | 30000 | >=0 | 1000 |
```